tests/decorators/test_branch_virtualenv.py

```diff
@@ -25,7 +25,11 @@
 pytestmark = pytest.mark.db_test
 
 
-class Test_BranchPythonDecoratedOperator:
+class Test_BranchPythonVirtualenvDecoratedOperator:
```

Comment:
Thanks @potiuk for fixing my copy & paste errors!

Reply:
:D => there were no errors, just inconveniences :D